### DISCUSSION TITLE
Where to chat about lingui?

### DISCUSSION THREAD

**Discussion by Bertg** [lingui/js-lingui#1240] [General]
The documentation is a bit unclear as to what the correct channels are to talk about Lingui.

* Spectrum: More than a year ago. https://spectrum.chat/linguijs?tab=posts
* Gitter: Over 5 months ago. https://gitter.im/lingui/js-lingui

And then there is this channel, GitHub Discussions, that isn't mentioned anywhere, but does seem to be actively used.

Should this info be consolidated, and cleaned up? Maybe even consider to bring up a new channel (many open source tools find themselves on Discord these days)?

If there is consensus on what the should be in the readme/site, I'll gladly make a PR with the fixes.

**Comment by semoal** [comment]
Hey @Bertg, you're right that we should update that.

We support Github Discussions where I'll try to answer as much as I can and just created **an official Discord for Lingui,** where we all can share our projects, questions, or improvements.

Invite: https://discord.gg/gFWwAYnMtA

If you can create a pull request updating this information, will be super appreciated 

Kind regards

**Reply by Bertg** [reply]
I will. Should I leave the StackOverflow links in there? Or just replace them with the GH discussions?

**Reply by Bertg** [reply]
In any case, Here is already the Spectrum/Gitter replacement with Discord. https://github.com/lingui/js-lingui/pull/1243
